pkgs/development/python-modules/bokeh/default.nix

```diff
@@ -55,13 +95,14 @@ buildPythonPackage rec {
     tornado
     numpy
     packaging
+    typing-extensions
   ]
   ++ lib.optionals ( isPy27 ) [
     futures
   ];
 
   checkPhase = ''
-    ${python.interpreter} -m unittest discover -s bokeh/tests
+    pytest tests
```

Comment:
i see you declared `disabledTests` did you mean to use `pytestCheckHook`?

Reply:
yea I was initially but since I needed to debug more that wasn't helpful